corehq/apps/app_manager/views/utils.py

```diff
@@ -419,12 +419,18 @@ def get_multimedia_sizes_for_build(domain, build_id, build_profile_id=None):
     build = get_app_cached(domain, build_id)
     assert build.copy_of, _("Size calculation available only for builds")
     build_profile = build.build_profiles[build_profile_id] if build_profile_id else None
-    # path: HQMediaMapItem object
-    multimedia_map_for_build = build.multimedia_map_for_build(build_profile=build_profile)
-    # path: CommCareMultimedia object
-    media_objects = dict(build.get_media_objects(multimedia_map=multimedia_map_for_build))
+    multimedia_map_for_build = {
+        media_map_item['multimedia_id']: media_map_item
+        for path, media_map_item in
+        build.multimedia_map_for_build(build_profile=build_profile).items()
+    }
+    media_objects = {
+        mm_object.get_id: mm_object
+        for path, mm_object in
+        build.get_media_objects(multimedia_map=multimedia_map_for_build)
```

Comment:
Since `get_media_objects` is expecting `media_map` to be keyed by `path`, it's weird to pass in a dict keyed by `multimedia_id`...it works but isn't guaranteed to keep working. Won't this work the same way if you keep directly passing in the output of `multimedia_map_for_build`, like before?

Reply:
Sorry, i meant to use the original structure with path as the key, i will update

Reply:
https://github.com/dimagi/commcare-hq/pull/25864/commits/8c95f3402f039e0f5ac2374c079de6bdcd610f67